compiler/llvm/llvmGlobalToWide.cpp

```diff
@@ -155,6 +156,28 @@ namespace {
 #endif
   }
 
+  template <typename BaseTy, std::enable_if_t<std::is_base_of_v<Function, BaseTy> ||
```

Comment:
These two look pretty fancy in terms of C++ features used. To help maintenance, could you add a comment to each describing what the purpose is & what the C++ template stuff is supposed to be doing?